Tests/Service/Request/RequestFactoryTest.php

```diff
@@ -245,6 +247,16 @@ public function testBuildFlowWithQueryParams()
             ->willReturn($originParamValue)
         ;
 
+        $serviceRequest->expects($this->once())
+            ->method('getArrParam')
+            ->willReturn($originArrParamValue)
+        ;
+
+        $serviceRequest->expects($this->once())
+            ->method('getDotParam')
```

Comment:
Need documentation for this.
Before it was obvious: `first-param` => `getFirstParam`, `arr-param` => `getArrParam`

Now: `dot.seprate[0].array` => `getDotParam`

Reply:
> Before it was obvious: first-param => getFirstParam, arr-param => getArrParam

Only in the test data. You could (and can) also have:
```
blah-blah => getFilter, whatEver => getName
```